llvm/test/tools/obj2yaml/XCOFF/aux-aligntype.yaml

```diff
@@ -0,0 +1,209 @@
+## Check that obj2yaml can parse SymbolAlignmentAndType, SymbolAlignment, and SymbolType.
+
+# RUN: yaml2obj %s --docnum=1 -o %t32
+# RUN: obj2yaml %t32 | FileCheck %s --check-prefix=CHECK32
+
+# CHECK32:      --- !XCOFF
```

Comment:
This test is about specific fields. You don't need to check the rest of the output; just enough to show that the field values are correct and are referring to the right symbol.